
See the comments in preserve_libkvm_abi.diff for an explanation

--- a/lib/libkvm/kvm_proc.c
+++ b/lib/libkvm/kvm_proc.c
@@ -720,6 +720,10 @@
 	ssize_t amount;
 	int fd;
 
+#if __FreeBSD_version >= 1000000
+#error "kvm_uread is not present in FreeBSD 10 ABI. Please remove preserve_libkvm_abi.diff"
+#endif
+
 	if (!ISALIVE(kd)) {
 		_kvm_err(kd, kd->program,
 		    "cannot read user space from dead kernel");
